src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import classnames from 'classnames';
+import { findTab } from '../../functions';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = findTab(tabs, selectedTabId);
+
+  const isActive = ({ id }) => selectedTab.id === id;
```

Comment:
The prefix `is` is for variables, not for functions. Maybe `checkIsActive` or something similar would better describe your function.